config/content.d/staging.horse.json

```diff
@@ -1,10 +1,8 @@
 {
   "staging.horse": {
     "content": {
-      "/docs-private-cloud/eng-handbook/": "https://github.com/rackerlabs/docs-rpc/handbook/docs/",
-      "/docs/private-cloud/rpc/master/": "https://github.com/rackerlabs/docs-rpc/master/",
```

Comment:
I think I might be missing something. Don't we want staging to build the master branch?